content/docs/studio/user-guide/projects-and-experiments/explore-ml-experiments.md

```diff
@@ -38,12 +38,17 @@ The branches and commits in your Git repository are displayed along with the
 corresponding models, metrics, hyperparameters, and DVC-tracked files.
 
 [New experiments submitted from Iterative Studio][run-experiments] appear as
-experiment commits, which are eventually pushed to Git. Any live metrics that
-you send using [DVCLive] are displayed in a special experiment row nested under
-the parent Git commit. More details of how live metrics are displayed can be
-found
+experiment commits, which are eventually pushed to Git. Experiments that you
+push using the `dvc exp push` command as well as any live metrics that you send
+using [DVCLive] are displayed in a special experiment row nested under the
+parent Git commit. More details of how live metrics are displayed can be found
 [here](/doc/studio/user-guide/projects-and-experiments/live-metrics-and-plots#view-live-metrics-and-plots).
 
+To manually check for updates in your repository, use the `Reload` button 🔄
+located above the project table.
+
+<!-- To do: Replace the following image with one that contains dvc exp and live experiment rows.-->
```

Comment:
@tapadipti Do you plan to replace the image in this PR or in a follow-up?

Reply:
In a follow-up PR.